src/components/converter/Callout.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import styled from 'styled-components'
+import { breakpoint } from '../../microsite-logic'
+import { PREACTIVATION_END } from '../../utils'
+import Timer from '../Timer'
+
+const medium = css => breakpoint('medium', css)
+
+const Callout = () => {
+  const preactivationActive = new Date() < PREACTIVATION_END
```

Comment:
Same here, should we use something like [`useNow()`](https://github.com/aragon/aragon/blob/e92dfb9c36d692edfb9e9dd6beab675a6680e5fd/src/hooks.js#L18-L30)?

Reply:
Done in 3b7b124.